apis/contour/v1beta1/tlscertificatedelegation.go

```diff
@@ -42,6 +42,8 @@ type CertificateDelegation struct {
 
 // TLSCertificateDelegation is an TLS Certificate Delegation CRD specificiation.
 // See design/tls-certificate-delegation.md for details.
+// +k8s:openapi-gen=true
+// +kubebuilder:resource:path=tlscertificatedelegations,singular=tlscertificatedelegation
```

Comment:
Which change ends up generating the "scope" field?

I guess I was expecting you'd have to add the "scope" label to kubebuilder:resource :)

Reply:
Thank you for the review!

`+kubebuilder:resource` marker generates the "scope" field.
If this marker exists, the kubebuilder seems to specify the `scope`'s default value (`Namespaced`).
However, it is good to specify `scope = Namespaced` in this marker. So I will correct it.

"k8s:openapi-gen" was specified in other CRDs, so I added it together. 

By the way, `contour/v1beta1/tlscertificatedelegation.go` and `projectcontour/v1/tlscertificatedelegation.go` still have the following differences, should I unify them?

```
$ diff -u apis/contour/v1beta1/tlscertificatedelegation.go apis/projectcontour/v1/tlscertificatedelegation.go
--- apis/contour/v1beta1/tlscertificatedelegation.go    2019-12-12 23:05:33.496594703 +0000
+++ apis/projectcontour/v1/tlscertificatedelegation.go  2019-12-12 23:03:15.595361314 +0000
@@ -43,7 +43,7 @@
 // TLSCertificateDelegation is an TLS Certificate Delegation CRD specificiation.
 // See design/tls-certificate-delegation.md for details.
 // +k8s:openapi-gen=true
-// +kubebuilder:resource:path=tlscertificatedelegations,scope=Namespaced,singular=tlscertificatedelegation
+// +kubebuilder:resource:path=tlscertificatedelegations,scope=Namespaced,shortName=tlscerts,singular=tlscertificatedelegation
 type TLSCertificateDelegation struct {
        metav1.TypeMeta   `json:",inline"`
        metav1.ObjectMeta `json:"metadata"`

```

Reply:
The CRDs in `contour/v1beta1` are deprecate, please leave them as they are.

Reply:
OK. I just specified the scope by kubebuilder's maker :)

And I changed the `check-stole` rule (in Makefile) a little for checking the auto-generated files of `examples/render` and `examples/contour` at once.

Reply:
What is the Makefile change needed for?